### PR TITLE
Update gradle

### PR DESCRIPTION
**Pull request by ggoldens**
This PR updates the version of the gradle build tools. In addition I'm upgrading the firebase and google play services version.

### CODE REVIEW

**Comment by ggoldens** [comment]
@marinaserranomontes or @msach22 could you please review this PR?

File: sample-app/build.gradle [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
     defaultConfig {
         applicationId "com.tokbox.android.IBSample"
         minSdkVersion 16
-        targetSdkVersion 26
-        versionCode 1
-        versionName "2.0.0"
+        targetSdkVersion 28
+        versionCode 2
+        versionName "3.0.0"
```
Comment (marinaserranomontes) [inline sample-app/build.gradle]: @ggoldens should it be 3.0.0? what is the semantic versioning we were using for IBS?

File: IB/build.gradle [hunk 2]
```diff
@@ -91,16 +91,16 @@ android {
...
-    compile 'com.squareup.picasso:picasso:2.5.2'
-    compile 'com.android.volley:volley:1.0.0'
-    compile 'com.google.firebase:firebase-database:10.0.1'
-    compile 'com.google.firebase:firebase-storage:10.0.1'
-    compile 'com.google.firebase:firebase-auth:10.0.1'
-    compile 'com.google.gms:google-services:3.1.0'
-    testCompile 'junit:junit:4.12'
+    compileOnly fileTree(include: ['*.jar'], dir: 'libs')
+    implementation 'com.android.support:appcompat-v7:28.0.0'
+    implementation 'com.opentok.android:opentok-android-sdk:2.14.2'
```
Comment (marinaserranomontes) [inline IB/build.gradle]: @ggoldens maybe, we could update to the latest (2.15.0 - October 2018)

Comment (ggoldens) [reply IB/build.gradle]: 👍 fixed

File: sample-app/build.gradle (hunk 1 above)
Comment (ggoldens) [reply sample-app/build.gradle]: You're right, I changed it, could you please 2x check?

**Comment by marinaserranomontes** [comment]
LGTM

**Pull request merged**